src/YesSql.Provider.MySql/MySqlCommandInterpreter.cs

```diff
@@ -61,5 +63,34 @@ public override void Run(StringBuilder builder, IAlterColumnCommand command)
                 result.Add(builder2.ToString());
             }
         }
+
+        public override void Run(StringBuilder builder, IAddIndexCommand command)
+        {
+            builder.AppendFormat("create index {1} on {0} ({2}) ",
+                _dialect.QuoteForTableName(command.Name, _configuration.Schema),
+                _dialect.QuoteForColumnName(command.IndexName),
+                string.Join(", ", command.ColumnNames.Select(x => GetColumName(x)).ToArray())
+                );
+        }
+
+        private static readonly Regex ColumnNamePattern = new(@"([a-zA-Z_][a-zA-Z0-9_]+)\s*(\(\s*(\d+)\s*\))?");
```

Comment:
Just split the string with `'(', ')', ' '` with the `RemoveEmptyEntries` options, and take the second result. I think Regexes are overkill here (and prone to issues).

Reply:
@sebastienros done. 

Let me know about merging and releasing